extension/manifest.json

```diff
@@ -6,7 +6,9 @@
   "author": "Federico Brigante",
   "default_locale": "en",
   "permissions": [
-    "storage"
+    "storage",
+    "cookies",
```

Comment:
Yes. I thought that working with cookies in an active session is not possible because of security policies. But it turned out that I just forgot about the fetch option. =)